File: C/2020/56限/array3.c
//  array3.c
//  cl  array3.c
#include <stdio.h>

int main() {
    int i[3][4] = {
        {1, 2, 3, 4},
        {5, 6, 7, 8},
        {11,22,33,44},
    };
    //
    for(int j = 0; j < 3; j ++) {
        for(int k = 0; k < 4; k ++) {
            printf("%d ", i[j][k]);
        }
        printf("\n");
    }

    return 0;
}

File: WebServer/2022/EarlyPeriod/array3.php
<?php   // array3.php
declare(strict_types=1);

//
$awk = [];
//
array_push($awk, 1);
var_dump($awk);
//
$awk[] = 2;
var_dump($awk);

//
echo "<br>";
$awk = [1, 2, 3, 4, 5, 6];
$pop = array_pop($awk);
var_dump($pop, $awk);

echo "<br>";
$awk = [1, 2, 3, 4, 5, 6];
$shift = array_shift($awk);
var_dump($shift, $awk);
